backend/api/prompts.py

```
@@ -0,0 +1,120 @@
+    client = AsyncOpenAI(api_key=api_key, base_url=base_url)
+    try:
+        response = await client.chat.completions.create(
+            model=settings.OPENAI_MODEL,
+            messages=[
+                {"role": "user", "content": prompt_text}
+            ],
+            temperature=0.0
+        )
+        content = response.choices[0].message.content
+        return {"result": content if content else ""}
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)